### PR TITLE
extend EPoS bound to 35%

### PR DESCRIPTION
**Pull request by rlan35**
As requested in the passed governance proposal: https://governance.harmony.one/#/staking-mainnet/proposal/QmdmnVwW6ob5UfB9hNYxKsRXAyM52jQpNXsV5Vw4fNbiqa

### CODE REVIEW

**Review by mindstyle85 — APPROVED** [review]
looks good, so as far as i understand code, you added a new variable CV2 which is 35%, then add or substract that from 1 to get the 1.35 and 0.65 bounds.

File: staking/effective/calculate.go [hunk 1]
```diff
@@ -148,13 +145,24 @@ func Compute(
...
+		picks[i].EPoSStake = effectiveStake(min, max, picks[i].RawStake)
 	}
 
 	return median, picks
 }
+
+func effectiveStake(min, max, actual numeric.Dec) numeric.Dec {
+	newMax := numeric.MinDec(max, actual)
+	return numeric.MaxDec(newMax, min)
+}
```
Comment (AlexiaChen) [inline staking/effective/calculate.go:168]: I noticed that the function was just refactored, the algorithm didn't actually change, just the max and min calculations were extracted externally, right? This does make it clearer

**Review by AlexiaChen — APPROVED** [review]
LGTM

File: staking/effective/calculate.go (hunk 1 above)
Comment (rlan35) [reply staking/effective/calculate.go:168]: yes correct.

**Pull request merged**